Makefile

```diff
@@ -125,23 +128,28 @@ start-integration: web/src/build/static/app.js bin/goalert bin/psql-lite bin/wai
 jest: node_modules 
 	yarn workspace goalert-web run jest $(JEST_ARGS)
 
-test: node_modules jest
+test: node_modules jest ## Run all unit tests
 	go test -short ./...
 
 force-yarn:
 	yarn install --no-progress --silent --frozen-lockfile --check-files
 
-check: force-yarn generate node_modules
-	# go run ./devtools/ordermigrations -check
-	go vet ./...
-	go run github.com/gordonklaus/ineffassign ./...
-	CGO_ENABLED=0 go run honnef.co/go/tools/cmd/staticcheck ./...
+check: check-go check-js  ## Run all lint checks
+
+check-js: force-yarn generate node_modules
 	yarn run fmt
 	yarn run lint
 	yarn workspaces run check
 	./devtools/ci/tasks/scripts/codecheck.sh
 
-check-all: check test smoketest cy-wide-prod-run cy-mobile-prod-run
+check-go:
+	go vet ./...
+	go fmt ./...
+	# go run ./devtools/ordermigrations -check
+	go run github.com/gordonklaus/ineffassign ./...
+	CGO_ENABLED=0 go run honnef.co/go/tools/cmd/staticcheck ./...
+
+check-all: check lint test smoketest cy-wide-prod-run cy-mobile-prod-run ## Run all lint checks and integration tests
```

Comment:
```suggestion
check-all: check test smoketest cy-wide-prod-run cy-mobile-prod-run ## Run all lint checks and integration tests
```
<img width="495" alt="Screen Shot 2022-10-04 at 10 15 32 AM" src="https://user-images.githubusercontent.com/38022260/193860859-f7842478-5d02-407c-9d82-bbd31aa6aba2.png">

Looks like a `lint` snuck in here